test/bdd/pkg/context/context.go

```diff
@@ -27,6 +27,6 @@ func NewBDDContext(caCertPath string) (*BDDContext, error) {
 
 	return &BDDContext{
 		TLSConfig:   &tls.Config{RootCAs: rootCAs, MinVersion: tls.VersionTLS12},
-		SidetreeURL: "http://localhost:48326/sidetree/v1/",
+		SidetreeURL: "https://localhost:443/sidetree/v1/",
```

Comment:
i wouldn't used default ssl port, suggestion: `9443`

Reply:
We need 443 for orb vdr

Reply:
done change to 9443